src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar container__item calendar--month-length--31">
```

Comment:
container__item it's redundant class. Please move style rules form it to calendar class